openapi-cli/src/test/resources/generators/service/ballerina/query/object_query.bal

```diff
@@ -0,0 +1,50 @@
+import ballerina/http;
+
+listener http:Listener ep0 = new (443, config = {host: "virtserver.swaggerhub.com"});
+
+service /lnash94/QueryParam/'1\.0\.0 on ep0 {
+    # searches inventory
+    #
+    # + required_query - 1. required query param with inline object
+    # + optional_query - 2. optional query param with inline object
+    # + default_query - 3. default query param with inline object
+    # + reference_query - 4. reference query param
+    # + required_nullable - 5. required query param with inline object and nullable true
+    # + optional_nullable - 6. optional query param with inline object
+    # + default_nullable - 7. default query param with inline object
+    # + reference_nullable - 8. reference_nullable query param won't consider nullable details when it has reference in swagger
+    # + return - returns can be any of following types
+    # http:Ok (search results matching criteria)
+    # http:BadRequest (bad input parameter)
+    resource function get inventory(Required_query required_query, Optional_query? optional_query, Manufacturer? reference_query, Required_nullable? required_nullable, Optional_nullable? optional_nullable, Manufacturer? reference_nullable, Default_query default_query = {"limit":"9","id":9}, Default_nullable default_nullable = {"limit":"9","id":9}) returns http:Ok|http:BadRequest {
```

Comment:
`Default_nullable default_nullable = {"limit":"9","id":9}` Will giving default record work in a scenario where there is a nested record ?